addons/account/wizard/account_report_tax.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class AccountTaxReport(models.TransientModel):
+    _inherit = "account.common.account.report"
+    _name = 'account.tax.report'
+    _description = 'Tax Report'
+
+    options = fields.Selection([('accrual_basis', 'Accrual Basis Method'), ('cash_basis', 'Cash Basis Method')], default='accrual_basis', string='Options')
```

Comment:
you don't need the option to select accrual/cash basis in this report. Remove it and compute everything like in accrual